esp32-led-blink-sdk/README.md

```diff
@@ -0,0 +1,39 @@
+# esp32-led-strip-sdk
+
+This example demonstrates how to integrate with the ESP-IDF SDK via CMake and how to use the standard GPIO library to control LED from Swift. This example is specifically made for the RISC-V MCUs from ESP32 (the Xtensa MCUs are not currently supported by Swift).
+
+![Led on](https://github.com/allexoK/swift-embedded-examples/assets/50021377/c40d4b3a-670b-4743-a8ae-a90c90b905b4)
```

Comment:
These links appear to point out of tree, could you fix them?

Reply:
I created the pics directory in the project folder and uploaded the pictures there, is it fine?